.github/workflows/ci.yml

```diff
@@ -24,15 +24,37 @@ jobs:
       - name: Checkout the repository
         uses: actions/checkout@v2
 
-      - name: Set up Python
+      - name: Setup Python
         uses: actions/setup-python@v2
         with:
           python-version: ${{ matrix.python }}
 
-      - name: Install Poetry
+      - name: Cache Poetry
+        id: cache-poetry
+        uses: actions/cache@v2
+        with:
+          path: ~/.poetry
+          key: poetry
```

Comment:
I think this will make poetry stuck in a single version right? That shouldn't happen. I think installing poetry doesn't take a considerable amount of time right?